### PR TITLE
SCMOD-8730: Added expiration policy

### PR DESCRIPTION
**Pull request by xbreizh**
https://portal.digitalsafe.net/browse/SCMOD-8730

### CODE REVIEW

File: job-service-contract/src/main/resources/com/hpe/caf/services/job/swagger.yaml [hunk 1]
```diff
@@ -480,14 +544,15 @@ definitions:
...
   failure:
     type: object
     properties:
       failureId:
         type: string
       failureTime:
-        type: integer
-        format: int64
+        type: string
+        format: date-time
```
Comment (dermot-hardy) [inline job-service-contract/src/main/resources/com/hpe/caf/services/job/swagger.yaml]: Failure time changing feels like a breaking change.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]
Cut this down to just the changes required to add expiration policy.  Do not try to align the contract with what is in Job Service (not for this PR anyway).

**Comment by xbreizh** [comment]
> Cut this down to just the changes required to add expiration policy. Do not try to align the contract with what is in Job Service (not for this PR anyway).

Done

**Comment by dermot-hardy** [comment]
Remove the unnecessary whitespace changes also.

**Pull request closed without merging**